.github/workflows/node.js.yml

```diff
@@ -15,7 +15,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [20.x]
+        node-version: [22.x]
```

Comment:
I updated this when debugging a CI lint test failure. The issue has been resolved and it's unrelated to this. However, I think it's better to upgrade to 22 since it's the LTS version.